### PR TITLE
60-Min Blitz- Autograd: Automatic Differentiation

### PR DESCRIPTION
**Pull request by CamiWilliams**
This update is in reference to the PyTorch Tutorial update strategy.

- Addressed issues #777 , #778 , #779 , #780 
- Gave more context to gradient descent, back propagation, automatic differentiation
- Added new code blog to explain gradient functions

### CODE REVIEW

File: beginner_source/blitz/autograd_tutorial.py [hunk 1]
```diff
@@ -3,78 +3,113 @@
...
 ###############################################################
-# Do a tensor operation:
+# Now, perform a simple tensor operation:
+
 y = x + 2
 print(y)
 
 ###############################################################
-# ``y`` was created as a result of an operation, so it has a ``grad_fn``.
+# ``y`` was created as a result of an operation, so it has a ``grad_fn`` that tells us its history.
```
[the remark this answers is not in the document]
Comment (CamiWilliams) [reply beginner_source/blitz/autograd_tutorial.py]: Done
